src/processgraph/datacube.js

```diff
@@ -217,7 +225,7 @@ module.exports = class DataCube {
 	}
 
 	getBands() {
-		return this.dimBands().getValues();
+		return this.dimBands().values;
```

Comment:
Why this change?

Reply:
Mmh, strange, now it works again with getValues(). Before I always got an error using it. I have reverted my change

Reply:
I applied the changes with the GEE naming convention and now this error pops up again:

`TypeError: this.dimBands(...).getValues is not a function`

Using values as before works fine.

Reply:
Okay, then let's try to fix that. I guess dimBands() returns null if there's no band dimension and then it fails to call getValues on null. Can you confirm which value is returned by dimBands()?

Reply:
![image](https://user-images.githubusercontent.com/15573047/58702323-c6d1ee00-83a5-11e9-8e0f-dbfdd764669d.png)

Reply:
Oh, wait. Why's there no getValues() in the Dimension class? Seems I missed to implement it. Could you simply add one? :-)